cmd/auth-handler.go

```diff
@@ -154,6 +154,13 @@ func validateAdminSignature(ctx context.Context, r *http.Request, region string)
 		return cred, owner, s3Err
 	}
 
+	logger.GetReqInfo(ctx).Cred = auth.Credentials{
+		AccessKey:  cred.AccessKey,
+		ParentUser: cred.ParentUser,
+	}
+	logger.GetReqInfo(ctx).Owner = owner
+	logger.GetReqInfo(ctx).Region = region
```

Comment:
```suggestion
	if ri := logger.GetReqInfo(ctx); ri != nil {
		ri.Cred = auth.Credentials{
			AccessKey:  cred.AccessKey,
			ParentUser: cred.ParentUser,
		}
		ri.Owner = owner
		ri.Region = region
	}
```

Reply:
Others need a similar treatment if you care.

Reply:
nice catch, added this check on all three locations